packages/app/components/stackbox/Stackbox.tsx

```diff
@@ -475,7 +475,7 @@ export const Stackbox = () => {
             </div>
             <div className="space-y-1">
               <div className="flex flex-col border divide-y md:divide-y-0 md:flex-row rounded-2xl border-surface-50 md:divide-x divide-surface-50">
-                <div className="flex flex-col w-full p-3 space-y-2 hover:bg-surface-25">
+                <div className="flex flex-col w-full pl-4 pr-3 py-2 lg:py-3 space-y-2 hover:bg-surface-25">
```

Comment:
This changes make the DatePicker input strange on mobile and tablet. With different padding from the other input.

Reply:
what's the reason to change `padding` here?

Reply:
It was requested in the ticket, it was one of the two updates to be done requested by Akash and prioritized by Venky
https://linear.app/swaprhq/issue/STK-188/cosmetic-fixes